src/mon.c

```diff
@@ -2152,10 +2152,14 @@ register struct monst *mtmp;
         struct obj* next;
         if (list) {
             if (Blind && !Deaf)
+                pline("A voice echoes in the arena:");
```

Comment:
Don't Repeat Yourself - this shouldn't be duplicated.

Reply:
thanks! i tried a more complicated-looking `if !deaf` followed by `if !blind` to separate out the things that rely on those senses, it looks messier to my eyes but at least it's not duplicated!